moped-editor/src/components/GridTable/Filters.js

```diff
@@ -481,33 +499,18 @@ const Filters = ({
                     (shouldRenderAutocompleteInput(
                       lookupTable,
                       operator,
-                      loading
+                      loading,
+                      lookupOperators
                     ) ? (
                       <Autocomplete
+                        freeSolo={shouldUseFreeSolo(operator)}
                         value={value || null}
-                        options={data[lookupTable]}
+                        options={data[lookupTable].map((option) =>
+                          getOptionLabel(option)
+                        )}
                         disabled={!filterParameters[filterIndex].operator}
-                        getOptionLabel={(option) =>
```

Comment:
A lot of these blocks of code had to do with determining what to do if a `field_name` was set in `PROJECT_LIST_VIEW_FILTERS_CONFIG` or not. The `getOptionLabel` function that is specific to each field now takes care of this.